jupyterlab_caip_optimizer/src/components/create_study.tsx

```diff
@@ -36,14 +38,14 @@ interface Props {
 
 const mapDispatchToProps = dispatch => ({
   createStudyAndLoad: (study: Types.Study) =>
-    dispatch(createStudy(study)).then((study: Types.Study) =>
+    dispatch(createStudy(study)).then((action: PayloadAction<Types.Study>) => {
```

Comment:
Should we leave a comment on this Redux-specific behavior? I feel like other people new to Redux might be confused about this part

Reply:
Yeah, redux's `createAsyncThunk` returns a `Promise<PayloadAction<type>>` since the action has more information. Read more here: https://redux-toolkit.js.org/api/createAsyncThunk#return-value